haystack/pipelines/standard_pipelines.py

```diff
@@ -217,6 +217,65 @@ def eval(
         )
         return output
 
+    def eval_batch(
+        self,
+        labels: List[MultiLabel],
+        params: Optional[dict] = None,
+        sas_model_name_or_path: Optional[str] = None,
+        sas_batch_size: int = 32,
+        sas_use_gpu: bool = True,
```

Comment:
I traced the `sas_use_gpu` parameter being passed to CrossEncoder via `semantic_answer_similarity` method. Let's keep in mind that we should soon replace all `use_gpu` parameter with `devices` parameter (as per https://github.com/deepset-ai/haystack/pull/3062 and https://github.com/deepset-ai/haystack/issues/2826) Just to keep in mind as todo item.